lib/tools/common/patching_parallel.py

```
@@ -0,0 +1,1244 @@
+            )
+
+        # STEP 6: Export commit as patch (same as serial)
+        rewritten_patch = None
+        if rewrite_patches and not is_autogen_dir and commit_hash:
+            try:
+                log.debug(f"About to export commit {commit_hash} for patch {patch_id} (patch_index={item.patch_index})")
+                rewritten_patch = export_commit_as_patch(item.mount_path, commit_hash)
+                # Log first few characters of the exported patch for debugging
+                preview = rewritten_patch[:200] if rewritten_patch else "None"
+                log.debug(f"Exported commit {commit_hash} as normalized patch for {patch_id}. Preview: {preview}")
+            except Exception as export_error:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)